src/Misc/layoutbin/en-US/strings.json

```diff
@@ -300,6 +300,7 @@
   "GCUnusedTrackingFile": "Mark tracking file '{0}' for GC, since it hasn't been used for {1} days.",
   "GenerateAndRunUpdateScript": "Generate and execute update script.",
   "GetSources": "Get Sources",
+  "GrantingFilePermissions": "Granting file permissions may take some time.",
```

Comment:
what about just `Granting file permissions to '{0}'`, {0} will be the account name?